src/plugins/intel_cpu/src/dnnl_postops_composer.cpp

```diff
@@ -50,7 +50,9 @@ DnnlPostOpsComposer::DnnlPostOpsComposer(const PostOps& postOps,
                                          const dnnl::memory::data_type outDataType,
                                          const std::vector<float>& legacyDqScales,
                                          bool useLegacyPostOps,
-                                         bool useLegacyZeroPoints)
+                                         bool useLegacyZeroPoints,
+                                         bool forceLegacyPostOps,
```

Comment:
There is DnnlPostOpsComposerLegacy to build legacy dnnl postops. What is the reason we cannot use that type in a context where we have to enforce the legacy behavior?

Reply:
Because in the current version we still have a need to force legacy post ops.
New version of the composer just has a different interface and works with `PostOps` data structure.
Let me replace those two flags with an enum, I planned to do it but forgot.

Reply:
Done